app/controllers/trainees/outcome_details_controller.rb

```diff
@@ -10,6 +10,12 @@ def recommended
       authorize trainee
     end
 
+    def recommend
```

Comment:
I generally try and avoid adding these custom actions. `QtsRecommendationsController#create` or something like that 🤷

Reply:
That would also be consistent with the way we're doing `TrnSubmissions`